Packs/MicrosoftGraphFiles/pack_metadata.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "Microsoft Graph Files",
-  "description": "Use the Microsoft Graph Files integration to enable your app get authorized access to files in OneDrive, SharePoint, and MS Teams across your entire organization. This integration requires admin consent.",
-  "support": "xsoar",
-  "currentVersion": "1.0.0",
-  "author": "Cortex XSOAR",
-  "url": "https://www.paloaltonetworks.com/cortex",
-  "email": "",
-  "created": "2020-04-14T00:00:00Z",
-  "categories": [
-    "Authentication"
-  ],
-  "tags": [],
-  "useCases": [],
-  "keywords": []
-}
+    "name": "Microsoft Graph Files",
+    "description": "Use the Microsoft Graph Files integration to enable your app get authorized access to files in OneDrive, SharePoint, and MS Teams across your entire organization. This integration requires admin consent.",
+    "support": "xsoar",
+    "currentVersion": "1.0.1",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "created": "2020-04-14T00:00:00Z",
+    "categories": [
+        "Authentication"
```

Comment:
please update here and in the integration yml the category

Reply:
will update to "IT Services"